app/src/main/java/comcmput301f16t01/github/carrier/RequestController.java

```diff
@@ -56,6 +56,9 @@ public void reset() {
     }
 
     public void cancelRequest(User rider, Request request) {
+        ElasticRequestController.CancelRequest cancelRequestTask = new ElasticRequestController.CancelRequest();
+        cancelRequestTask.execute(request);
+
```

Comment:
Again, thanks for starting cancelling a request in elastic search, but please test this as I an unsure (an cannot check myself) whether or not this works.